blackjack.el

```diff
@@ -1065,13 +1065,13 @@ Can be a single-character currency symbol such as \"$\", \"€\" or \"£\", or a
 (define-minor-mode blackjack-minor-mode
   "Blackjack minor mode.
 
-\\{blackjack-mode-map}"
+\\{blackjack-minor-mode-map}"
   :group 'blackjack
   :lighter " blackjack")
 
-(define-derived-mode blackjack-mode fundamental-mode "Blackjack"
+(define-derived-mode blackjack-mode special-mode "Blackjack"
   "Blackjack game mode."
-  (read-only-mode))
+  :group 'blackjack)
```

Comment:
I don't want the user to be able to modify the buffer, removing this would allow that.

Reply:
The user should indeed not be able to modify the buffer. However, this explicit call to read-only-mode is now redundant since special-mode does that for us. Try it!

Reply:
Everything seems to work.